src/components/textarea/textarea.stories.tsx

```diff
@@ -1,19 +1,98 @@
+import { Meta, StoryObj } from '@storybook/react'
 import { TextArea } from './index'
 
 export default {
   title: 'TextArea',
```

Comment:
Can you add the `Components` prefix here i.e. `Components/TextArea` please so it moves to the relevant folder in Storybook

Reply:
- [ ] Add `Components/` prefix to TextArea story title